models/optional_modules/consent/snowplow_web_consent_users.sql

```diff
@@ -0,0 +1,101 @@
+{{
+  config(
+    materialized='table',
+    sql_header=snowplow_utils.set_query_tag(var('snowplow__query_tag', 'snowplow_dbt'))
+  )
+}}
+
```

Comment:
In theory the log table should not become that massive as it should be a handful per user for a long period of time. However, there ARE unknowns on how this would actually work in real life so I could make it a native dbt incremental table just to be safe without a huge amount of effort on this table so that's fine.